common/djangoapps/student/views.py

```diff
@@ -1545,6 +1546,10 @@ def _do_create_account(form, custom_form=None):
 
     Note: this function is also used for creating test users.
     """
+    # feature implemented in edx
```

Comment:
can you please describe why you raise this error?

Reply:
because this is how edx does it. might be better to replace it with 404?

Reply:
I mean describe it in the comment to make it more clear, for example
`To disable a user creation, this part is taken from the future releases of the platform`, something like that))